vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -379,18 +379,7 @@ void Contribution::CreateNewEntry(
   if (wallet_balance < queue->amount) {
     contribution->amount = wallet_balance;
     queue->amount = queue->amount - wallet_balance;
-
-    if (queue->type == ledger::RewardsType::RECURRING_TIP ||
-        queue->type == ledger::RewardsType::ONE_TIME_TIP) {
-      AdjustPublisherListAmounts(
-          std::move(queue_publishers),
-          &publishers_new,
-          &publishers_left,
-          wallet_balance);
-      queue->publishers = std::move(publishers_left);
-    } else {
-      publishers_new = std::move(queue_publishers);
-    }
+    publishers_new = std::move(queue_publishers);
```

Comment:
It seems like in both branches we have `publishers_new = std::move(queue_publishers);`.

Maybe we don't need `publishers_new` anymore, then?

Also, looks like `publishers_new` is now unused.